fish-rust/src/ffi.rs

```diff
@@ -80,6 +81,12 @@ impl parser_t {
         let ffi_jobs = self.ffi_jobs();
         unsafe { slice::from_raw_parts(ffi_jobs.jobs, ffi_jobs.count) }
     }
+
+    pub fn get_libdata_pod(&mut self) -> &mut library_data_pod {
```

Comment:
nit: I could do without the `get_` prefix (there are no field/method conflicts in rust) but that's fine.
Looks good.